cmd/deploy/deploy.go

```diff
@@ -228,13 +225,10 @@ func (dc *DeployCommand) RunDeploy(ctx context.Context, deployOptions *Options)
 	os.Setenv(model.OktetoNameEnvVar, deployOptions.Name)
 
 	if deployOptions.Dependencies && !okteto.IsOkteto() {
-		return fmt.Errorf("deploy of dependencies is only available in clusters managed by Okteto")
+		return fmt.Errorf("'dependencies' is only available in clusters managed by Okteto")
```

Comment:
```
'dependencies' is only available when using a cluster deployed in Okteto Cloud
```

Would this be more clear? It might not be clear to new users what we mean by just using "Okteto".

Reply:
What I mean here is that `dependencies` is only available if you are in Okteto Cloud, Okteto for Teams, or Okteto self-hosted. But not in vanilla Kubernetes clusters (yet). 
What would you suggest then?